deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/SpecInputModel.java

```diff
@@ -3,12 +3,21 @@
 import static java.util.Objects.requireNonNull;
 
 import java.io.InputStream;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Objects;
 
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+import io.quarkiverse.openapi.generator.deployment.SpecConfig;
+import io.smallrye.config.PropertiesConfigSource;
+
 public class SpecInputModel {
 
     private final InputStream inputStream;
     private final String filename;
+    private final Map<String, String> codegenProperties = new HashMap<>();
```

Comment:
Should not this constructor call the new one using this reserved word?, passing null as the default package name

Reply:
Why? What's the difference? Every time this class is instantiated, this field will be instantiated. If you mean the second constructor added to this class, yes. I copied and pasted, I will remove and call this(param, param). Forgot to change.